defaults.env

```diff
@@ -3,3 +3,4 @@ PULSE_DOMAIN=www.mozillapulse.org
 NETWORK_API_DOMAIN=network.mofoprod.net
 VIRTUAL_ROOT=/
 TARGET_DOMAIN=network.mofostaging.net
+CSP=default-src 'self'; script-src 'self' cdn.optimizely.com; style-src 'self' code.cdn.mozilla.net fonts.googleapis.com; img-src * data:; font-src 'self' fonts.gstatic.com fonts.googleapis.com code.cdn.mozilla.net; connect-src 'self' *.mozilla.org; media-src 'self'; object-src 'self'; child-src 'self'; frame-src 'self';
```

Comment:
How about [this](https://cspvalidator.org/#headerValue%5B%5D=default-src+'self'%3B+script-src+'self'+cdn.optimizely.com%3B+style-src+'self'+code.cdn.mozilla.net+fonts.googleapis.com%3B+img-src+*+data%3A%3B+font-src+'self'+fonts.gstatic.com+fonts.googleapis.com+code.cdn.mozilla.net%3B+connect-src+'self'+*.mozilla.org%3B+media-src+'self'%3B+object-src+'self'%3B+frame-src+'self'%3B&strategy=intersection)?